packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -312,29 +314,31 @@ You can set functionPerRoute: false to prevent surpassing the limit.`
 	};
 }
 
-function getRuntime() {
+function getRuntime(verbose?: boolean) {
```

Comment:
nit: It would be nice to extract the verbose part as a new function called `validateRuntime` instead.